### PR TITLE
feat(prettier): support format function for ImportNamespaceSpecifier

### PR DESCRIPTION
**Pull request by Dunqing**

### CODE REVIEW

**Comment by Dunqing** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1441** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/1441?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #1442** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/1442?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #1443** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/1443?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/oxc-project/oxc/1443?utm_source=stack-comment).

File: crates/oxc_prettier/src/format/mod.rs [hunk 1]
```diff
@@ -1013,7 +1021,7 @@ impl<'a> Format<'a> for ImportAttribute {
 impl<'a> Format<'a> for ExportNamedDeclaration<'a> {
     fn format(&self, p: &mut Prettier<'a>) -> Doc<'a> {
         let mut parts = p.vec();
-        parts.push(module::print_module_specifiers(p, &self.specifiers, false));
+        parts.push(module::print_module_specifiers(p, &self.specifiers, false, false));
```
Comment (Boshen) [inline crates/oxc_prettier/src/format/mod.rs:1024]: When there are multiple boolean arguments like this, it is sometimes good to document it like this: `, /* is_default */ false, /* is_namespace */ true)` for readability.

**Comment by Boshen** [comment]
You may merge this after rebase.

**Comment by Boshen** [comment]
Oh, we need to restack this on top of main 😅

**Comment by Boshen** [comment]
Haha, I think you need `gt move` to move the stack.

**Comment by Dunqing** [comment]
> Haha, I think you need `gt move` to move the stack.

This is so weird

**Pull request merged**